packages/@aws-cdk/aws-cloudfront/test/web-distribution.test.ts

```diff
@@ -597,6 +600,49 @@ added the ellipsis so a user would know there was more to ...`,
     test.done();
   },
 
+  'distribution with CloudFront function-association'(test: Test) {
```

Comment:
Need this test for `Distribution` as well.

Reply:
There is an integ test for distribution. Is this ok or do you need the unit test too?

Reply:
A unit test too, please. Unit tests are run much more frequently and are easier to maintain. In my perspective, at least, everything should have a unit test to verify intended functionality, and each resource (or resource-link) an integration test to verify deployability.

Reply:
Done